src/pages/GameDetails.jsx

```diff
@@ -1,8 +1,49 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 import GameDetailsComponent from "../components/GameDetailsComponent";
+import authUser from "../utils/authUser";
+import { useNavigate } from "react-router-dom";
 
 function GameDetails() {
   const { gameId } = useParams();
+  const [data, setData] = useState({});
+  const [leaderBoards, setLeaderBoards] = useState([]);
+  const navigate = useNavigate();
+  const config = {
+    headers: {
+      Authorization: "Bearer " + localStorage.getItem("token"),
+    },
+  };
+
+  useEffect(() => {
+    axios
+      .get("/games/" + gameId, config)
+      .then((res) => {
+        setData(res.data);
+      })
+      .catch((err) => {
+        console.log(err.data);
+      });
+  }, []);
+
+  useEffect(() => {
+    axios
+      .get("/games/leaderboard/" + gameId, config)
+      .then((res) => {
+        setLeaderBoards(res.data?.data);
+      })
+      .catch((err) => {
+        console.log(err.data);
+      });
+  }, []);
+
+  const handlePlayingGame = () => {
+    let route = data.data?.id;
+    axios.post("/games/view-count/" + route).then((res) => {
```

Comment:
mas @masegy ini kayanya better play-count aja mas

nanti aku tambahin axios.post buat view-count ke tombol "play this game" yang ada di landing/listing (component GameBox)